### PR TITLE
Return correct color values on color proxies.

### PR DESCRIPTION
**Pull request by Yagira**

### CODE REVIEW

**Comment by thegrb93** [comment]
Why do experimental parents want 0-1 while not experimental is 0-255? Seems strange to me. The added conversions here will probably break a lot of outfits.

**Comment by Yagira** [comment]
> Why do experimental parents want 0-1 while not experimental is 0-255? Seems strange to me.

You'll probably have to ask caps. 

> The added conversions here will probably break a lot of outfits.

I suggested this change mostly because these proxies don't work "by default" and you have to manually tweak them in the expression. Some of them return 0-1 while others do 0-255 which seems really inconsistent and I think that it causes most inexperienced uses to just assume that the proxy doesn't work at all. While there is a chance for it to break existing outfits, I believe that people who know what they are doing could easily fix them.

**Comment by thegrb93** [comment]
I'll see what they think, if we should edit the parts themselves or if this is more suitable. Either way, outfits will break, but I agree that the current way is inconsistent and confusing.

File: lua/pac3/core/client/parts/proxy.lua [hunk 1]
```diff
@@ -908,6 +960,8 @@ function PART:RunExpression(ExpressionFunc)
 	return pcall(ExpressionFunc)
 end
 
+local oldclass = ""
```
Comment (roboderpy) [inline lua/pac3/core/client/parts/proxy.lua]: ?

Comment (thegrb93) [reply lua/pac3/core/client/parts/proxy.lua]: Used to only update the expression if the parent becomes a different class?

**Comment by thegrb93** [comment]
Yah good call

**Comment by thegrb93** [comment]
I'll take a look at the color parameters of the parts and see if there isn't a more elegant solution.

**Comment by thegrb93** [comment]
If the parameters in the editor are 0-1 I think the proxy should be that way too.

**Pull request closed without merging**